topics/plugin-preprocess-xslt.dita

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE task PUBLIC "-//OASIS//DTD DITA Task//EN" "task.dtd">
+<!--  This file is part of the DITA Open Toolkit project. See the accompanying LICENSE file for applicable license.  -->
+<task id="plugin-xsltparams">
+  <title>Processing topics with XSLT in preprocess</title>
+  <shortdesc>You can pass parameters from the Ant build to existing XSLT steps in both the pre-processing pipeline and
```

Comment:
Treat all the content as something that a Person Who Knows What They're Doing needs to go over and copy edit.